.settings.xml

```diff
@@ -1,60 +1,70 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
-  <servers>
-	<server>
-	  <id>repo.spring.io</id>
-	  <username>${env.CI_DEPLOY_USERNAME}</username>
-	  <password>${env.CI_DEPLOY_PASSWORD}</password>
-	</server>
-  </servers>
-  <profiles>
-	<profile>
-	  <id>spring</id>
-	  <activation><activeByDefault>true</activeByDefault></activation>
-	  <repositories>
-		<repository>
-			<id>spring-snapshots</id>
-			<name>Spring Snapshots</name>
-			<url>http://repo.spring.io/libs-snapshot-local</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-		</repository>
-		<repository>
-			<id>spring-milestones</id>
-			<name>Spring Milestones</name>
-			<url>http://repo.spring.io/libs-milestone-local</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</repository>
-		<repository>
-			<id>spring-releases</id>
-			<name>Spring Releases</name>
-			<url>http://repo.spring.io/release</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</repository>
-	  </repositories>
-	  <pluginRepositories>
-		<pluginRepository>
-			<id>spring-snapshots</id>
-			<name>Spring Snapshots</name>
-			<url>http://repo.spring.io/libs-snapshot-local</url>
-			<snapshots>
-				<enabled>true</enabled>
-			</snapshots>
-		</pluginRepository>
-		<pluginRepository>
-			<id>spring-milestones</id>
-			<name>Spring Milestones</name>
-			<url>http://repo.spring.io/libs-milestone-local</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</pluginRepository>
-	  </pluginRepositories>
-    </profile>
-  </profiles>
+	<servers>
+		<server>
+			<id>repo.spring.io</id>
+			<username>${env.CI_DEPLOY_USERNAME}</username>
+			<password>${env.CI_DEPLOY_PASSWORD}</password>
+		</server>
+	</servers>
+	<profiles>
+		<profile>
+			<id>spring</id>
```

Comment:
Add this boilerplate text 

```
      <!--
          N.B. this profile is only here to support users and IDEs that do not use Maven 3.3. 
          It isn't needed on the command line if you use the wrapper script (mvnw) or if you use 
          a native Maven with the right version. Eclipse users should points their Maven tooling to
          this settings file, or copy the profile into their ~/.m2/settings.xml.
      -->
```

Also, i don't think the `</servers>` section is needed but I haven't checked.